lightwood/data/timeseries_transform.py

```diff
@@ -14,7 +14,8 @@
 
 def transform_timeseries(
         data: pd.DataFrame, dtype_dict: Dict[str, str],
-        timeseries_settings: TimeseriesSettings, target: str, mode: str) -> pd.DataFrame:
+        timeseries_settings: TimeseriesSettings, target: str, mode: str,
+        use_dask: bool = True, mp_row_cutoff: int = 500) -> pd.DataFrame:
```

Comment:
Much better to default to `False` and require it to be `True` + dask installed. Otherwise, there's a risk in terms of version incompatibility + it's usually preferred to have consistent behavior regardless of user environment with a few critical exceptions (e.g. using GPU when we detect cuda)